### DISCUSSION TITLE
What are some cheap and efficient SBC/camera options that work with MediaMTX?

### DISCUSSION THREAD

**Discussion by Redsandro** [bluenviron/mediamtx#2118] [General]
I used a cheap $6 1080P camera module with a $18 Raspberry Pi Zero W 2 and a 3d printed case. It works beautifully as a cheap RTSP camera.

I've also tried a Banana Pi BPI-M2 Zero that's even cheaper. They use recycled memory components, which is interesting. But software support is poor. It's difficult to compile a working camera driver. And it's impossible to do any video encoding.

The Pi Zero W 2 runs pretty hot though, probably because of the GPU encoding H264, while it's only using a quarter of the CPU power. And the camera is really not that great. I don't need such a powerful SoC. I do need a better camera. QHD or UHD.

Is there a more interesting SBC + camera combo that runs Linux and MediaMTX? Trying to build one yourself really makes you appreciate the technical design of those cheap PTZ cameras from brands such as Xiaomi and Reolink. Too bad the software/firmware is a show stopper.

**Comment by dragulesq** [comment]
I have a similar setup: PI Zero W with Raspicam v2 (IMX219) and MediaMTX. I am also looking to upgrade both the camera (not great in low light) and board so I can have ethernet and possibly 4K HEVC encoding. I like the Orange PI Zero 3 specs, price and form factor but looks there is no kernel support for the VPU on that board.

[a comment by an automated account is not shown]
